extensions/exposition/features/cache.feature

```diff
@@ -0,0 +1,188 @@
+Feature: Caching
+
+  Background:
+    Given the `identity.basic` database contains:
+      # developer:secret
+      # user:12345
+      | _id                              | username  | password                                                     |
+      | efe3a65ebbee47ed95a73edd911ea328 | developer | $2b$10$ZRSKkgZoGnrcTNA5w5eCcu3pxDzdTduhteVYXcp56AaNcilNkwJ.O |
+    And the `identity.bans` database is empty
+
+  Scenario: Caching successful response
+    Given the annotation:
+      """yaml
+      /:
+        anonymous: true
+        GET:
+          cache:control: max-age=60000
+          dev:stub: hello
+      """
+    When the following request is received:
+      """
+      GET / HTTP/1.1
+      accept: text/plain
+      """
+    Then the following reply is sent:
+      """
+      200 OK
+      content-type: text/plain
+      cache-control: max-age=60000
+
+      hello
+      """
+
+  Scenario: Nested cache directives
+    Given the annotation:
+      """yaml
+      /:
+        cache:control: max-age=30000
+        GET:
+          anonymous: true
+          dev:stub: hello
+        /foo:
+          auth:role: developer
```

Comment:
Within these features, there is no step for adding a `developer` role to a user.

It passes for you because you're re-using the user `id` from other tests.

Reply:
We should add a hook to clear databases after each scenario later.